### PR TITLE
no more hacking on other's dependency code

### PR DESCRIPTION
**Pull request by tohjg**
after creating another new project with this library, i found error message saying 'unhandled command'. after few hours scratching my head, i found that i have to change `react-native-udp`'s code.

thus to make others' life easier in future, i created this pull request.

tested on 
rn 0.44.2
node 6.9.5
ios 10.3
android 5.1.1

### CODE REVIEW

**Comment by tohjg** [comment]
btw, the true implementation is at [another proper parsing without setting Buffer as global](https://github.com/netbeast/react-native-ssdp/pull/14/commits/43e19448c3f4dc51bb063a03f38809053ae643a8)

other's commits are my testing over npm

File: README.md [hunk 1]
```diff
@@ -17,12 +17,6 @@ npm install react-native-ssdp
 rnpm link
 ```
 
-Make sure you set `Buffer` as global in `react-native-udp`'s UdpSockets.js (noted filename end with letter s)
```
Comment (jsdario) [inline README.md:20]: Why is necessary to remove this from the docs?

**Comment by jsdario** [comment]
Hi, thanks for your PR.

I see the value of isolating Buffer from the global spec, but this is already mandatory from a peer dependency, so there is not a huge gain.

https://github.com/tradle/react-native-udp#note

It is also working on production, is there a particular bug you are experiencing that is solved by this PR?

**Comment by tohjg** [comment]
hi,

this PR is improvement in term of user experience. developer just `npm install react-native-ssdp` and its dependencies and it's ready to use 🎉🎉

we just can skip adding setting Buffer in `react-native-udp`'s library. yahooooo! 

Buffer is not mandatory. by default, when Buffer is not set inside `react-native-udp`, type of message that return from `react-native-udp` is uint8array. Thus, Buffer is declared in `react-native-ssdp` instead and wrapped against returned message as we need it to parse message to string.

I tried set Buffer globally in elsewhere in my source code but `react-native-udp` is not picking it up. I'm not sure it's react native problem or what not.

**Comment by tohjg** [comment]
btw, referring to the commit label [removed Buffer dependency as it’s old backward compatibility library](https://github.com/netbeast/react-native-ssdp/pull/14/commits/81a99e4f0283aa28c8db7e9f5561aa1eeaddaa5e) it's about performance consistency on parsing uint8array to string.

i compared it with Buffer. it could give consistency of 0.4-0.5ms while Buffer always get inconsistency parsing time, in between 0.4-1.2ms.

the bad side of it is that it has performance lose when parsing huge bytes.

**Pull request closed without merging**